djangocms_icon/fields.py

```diff
@@ -4,14 +4,25 @@
 from django.conf import settings
 from django.db import models
 from django.forms import fields, widgets
+from django.template.loader import render_to_string
 from django.utils.translation import ugettext
 
 
 def get_iconsets():
-    return getattr(settings, 'DJANGOCMS_ICON_SETS', (
-        ('fontawesome', 'fa', 'Font Awesome'),
+    iconsets = getattr(settings, 'DJANGOCMS_ICON_SETS', (
+        ('fontawesome5regular', 'far', 'Font Awesome 5 Regular', 'lastest'),
+        ('fontawesome5solid', 'fas', 'Font Awesome 5 Solid', 'lastest'),
+        ('fontawesome5brands', 'fab', 'Font Awesome 5 Brands', 'lastest'),
     ))
 
+    set = []
```

Comment:
Why are you overriding built-in `set`? Please change the name of this variable.

Reply:
Sorry for the mishap. I changed the variable name.